src/legacy/core_plugins/timelion/public/timelion_vis_fn.ts

```diff
@@ -43,11 +40,11 @@ export const timelionVis = () => ({
     interval: {
       types: ['string', 'null'],
       default: 'auto',
-    }
+    },
   },
-  async fn(context, args) {
+  async fn(context: any, args: any) {
```

Comment:
These can go away entirely once you use the `ExpressionFunction` generic, as it will already handle typing the params for you

Reply:
Yes. Got it. Done